doc/source/upgrading.rst

```diff
@@ -22,8 +22,9 @@ You can upgrade |PBM| to the **latest version** or to a **specific version**. Si
 .. rubric:: Important notes
  
 1. Backward compatibility between data backup and restore is supported for upgrades within one major version only (for example, from 1.1.x to 1.2.y). When you upgrade |PBM| over several major versions (for example, from 1.0.x to 1.2.y), we recommend to make a backup right after the upgrade.
-2. Starting from version 1.3.0, |pbm| packages are stored in the *pbm* repository and the *tools* repository for backward compatibility. To upgrade to the version lower than 1.3.0, enable the *tools* repository.
-3. Upgrade |PBM| on all nodes where it is installed.
+2. |PBM| v1.5.0 and later is incompatible with |PBM| v1.4.1 and earlier due to different processing of system collections ``Users`` and ``Roles`` during backup / restore operations. After the upgrade to |PBM| v1.5.0 and later, make sure to make a fresh backup.
+3. Starting from version 1.3.0, |pbm| packages are stored in the *pbm* repository and the *tools* repository for backward compatibility. To upgrade to the version lower than 1.3.0, enable the *tools* repository.
```

Comment:
I think we should remove this advice regarding how to find the packages for 1.2.x or lower.